test/pmmh.jl/pmmh.jl

```diff
@@ -0,0 +1,25 @@
+using Distributions
+using Turing
+using Base.Test
+srand(125)
+
+x = [1.5 2.0]
+
+@model pmmhtest(x) = begin
+  s ~ InverseGamma(2,3)
+  m ~ Normal(0,sqrt(s))
+  for i in 1:length(x)
+    x[i] ~ Normal(m, sqrt(s))
+  end
+  s, m
+end
+
+check_numerical(
```

Comment:
@emilemathieu Hello. Did you define this function `check_numerical` somewhere in Turing? It seems that I can't find it.

Reply:
Never mind - I found it in file `test/utility.jl`.